### ISSUE TITLE
`make bundle` fails with v1.18.0

### ISSUE THREAD

**Issue by umangachapagain** [operator-framework/operator-sdk#5587]
#### What did you do?
On a project with sdk v1.18.0 and project config version 3,

- used a workaround for  #5574 
- used a non default container name for manager container (it's manager by default)
- `make bundle` 

#### What did you expect to see?
A valid bundle consistent with bundle generated by older SDK for the same project.

#### What did you see instead? Under which circumstances?
Running `make bundle` failed with following error
```shell
Updating generated manifests
/home/uchapaga/go/bin/controller-gen "crd:trivialVersions=true" rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
operator-sdk generate kustomize manifests --interactive=false -q
cd config/manager && /home/uchapaga/go/bin/kustomize edit set image controller=quay.io/ocs-dev/ocs-operator:latest
/home/uchapaga/go/bin/kustomize build config/manifests | operator-sdk generate bundle 
Generating bundle
Generating bundle manifests
FATA[0000] Error generating bundle manifests: manager deployment does not have container named "manager" 
```

This forces us to use labels and container names defined by SDK v.18.0 which wasn't the case before.

#### Environment

**Operator type:**

/language go

**Kubernetes cluster type:**

OpenShift

```shell
$ operator-sdk version
operator-sdk version: "v1.18.0", commit: "c9c61b6921b29d731e64cd3cc33d268215fb3b25", kubernetes version: "1.21", go version: "go1.17.7", GOOS: "linux", GOARCH: "amd64"
```

```shell
$ go version
go version go1.17.1 linux/amd64
```

```shell
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v0.21.0-beta.1", GitCommit:"96e95cef877ba04872b88e4e2597eabb0174d182", GitTreeState:"clean", BuildDate:"2021-12-15T01:45:40Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by jmrodri** [comment]
@umangachapagain is this an existing project or is it a newly generated one with operator-sdk 1.18?

**Comment by umangachapagain** [comment]
> @umangachapagain is this an existing project or is it a newly generated one with operator-sdk 1.18?

@jmrodri it is an [existing project](https://github.com/red-hat-storage/ocs-operator) that was using sdk v1.2.0.

**Comment by jberkhahn** [comment]
this should be fixed by the solution we have coming for #5574. Look for that from @ryantking

**Comment by asmacdo** [comment]
@ryantking can we close this or #5574? they seem to be dupes

**Comment by umangachapagain** [comment]
@asmacdo These are 2 different issues, not duplicates. When you fix 5574, you will hit this issue as I mentioned in the bug report above.

**Comment by ryantking** [comment]
This is fixed in [v1.19.1](https://github.com/operator-framework/operator-sdk/releases/tag/v1.19.1).
